cmake/CMakeLists.txt

```diff
@@ -196,6 +196,9 @@ option(onnxruntime_ENABLE_ATEN "Enable ATen fallback" OFF)
 
 option(onnxruntime_BUILD_KERNEL_EXPLORER "Build Kernel Explorer for testing and profiling GPU kernels" OFF)
 
+# https://zeux.io/2010/11/22/z7-everything-old-is-new-again/
+cmake_dependent_option(MSVC_Z7_OVERRIDE "replacing /Zi and /ZI with /Z7 when using MSVC with CCache" ON "MSVC" OFF)
```

Comment:
If I don't specify use_cache when running build.py this is defaulting to ON for a Windows build isn't it? I would have expected that to be undesirable for a local build. 

Is there a reason this needs to be more complicated than simply defaulting to OFF so it has to be explicitly specified? Or if we really need to hide the option when MSVC is false, the default value could also be OFF.

Reply:
Good finding. Thank you very much. I've made a little improvement.
So far, I agree simply defaulting to OFF should work too. 
One weak reason, maybe, someone would build ORT with clang on Windows.